tests/Validator/LIVR/Rules/ListLengthTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Validator\LIVR\Rules;
+
+use PHPUnit\Framework\TestCase;
+use uMotif\LIVR;
+
+final class ListLengthTest extends TestCase
+{
+    private const TEST_DATA_KEY = 'my_rule_list';
+    private const FORMAT_ERROR = 'FORMAT_ERROR';
```

Comment:
Instead of referring to this constant you could refer to the constant set in the ListLength rule class.

Reply:
I wanted to keep it separate, as the tests should be completely independent from the _class-being-tested_.

Reply:
Yeah, you're right. Also, if the class constants changed value it would be better if the tests failed.